sass/components/_asset-card.scss

```diff
@@ -0,0 +1,107 @@
+.asset-card {
+    display: grid;
+    grid-template-areas:
+        "banner"
+        "desc"
+        "tags";
+    grid-template-rows: 12rem 5.5rem 2rem;
+
+    text-decoration: none;
+    overflow: hidden;
+    border-radius: $border-radius;
+    background-color: $syntax-theme-background;
+    color: $color-white;
+    border: 2px solid #383838;
+
+    &:hover {
+        background-color: $card-hover-background;
+        border: 2px solid #6b6b6b;
+    }
+
+    &__banner {
+        grid-area: banner;
+        display: block;
+        position: relative;
+
+        background: linear-gradient(45deg,

```

Comment:
Totally a matter of opinion, but I'd like to propose a more subtle / aligned gradient: `linear-gradient(0deg, #37393e 0%, #1b1b1b 100%)`

![image](https://user-images.githubusercontent.com/2694663/185258258-d8741f68-0156-4084-afd4-3c6526e3a9af.png)

The current one draws the eye a bit too much and feels a bit aggressive / like a bright light is shining on something very metallic / reflective. And notably with that analogy the light should behave differently for each tile based on its position relative to your eye, but it doesn't. Which also contributes to my (personal) impression that it feels "off":

![image](https://user-images.githubusercontent.com/2694663/185258410-747669b3-d8ee-4fb5-9be2-8674fd31a142.png)

Reply:
Good idea, I like it, a lot more subtle while still being nicer than a solid color.